File: uri/1013/1013.c
#include <stdio.h>
#include <stdlib.h> // abs
 
int maior(int, int);
 
int main() {
    int a, b, c;
    
    scanf ("%d %d %d", &a, &b, &c);
    printf ("%d eh o maior\n", maior(a, maior(b,c)));
 
    return 0;
}


int maior(int a, int b) {
    return (a + b + abs(a-b)) / 2;
}

File: uri/1013/1013.rb
def maior(a, b)
    (a + b + (a-b).abs) / 2
end

input = gets.chomp
a = input.scan(/\d+/).map{|x| x.to_i}

puts "#{maior(a[0], maior(a[1], a[2]))} eh o maior"
